server/service/logger.go

```diff
@@ -5,12 +5,16 @@ import (
 )
 
 // NewLogger returns a logrus Logger
-func NewLogger() logrus.FieldLogger {
+func NewLogger(env string) *logrus.Logger {
 	logger := logrus.New()
 	logger.Formatter = &logrus.TextFormatter{
 		FullTimestamp:   true,
 		TimestampFormat: "2006-01-02 15:04:05",
 	}
 
+	if env == "production" || env == "staging" {
```

Comment:
`production` behavior should be the default one.
It's better to override things when we're in development.

Reply:
I mimic `yarn/npm` behavior. But production by default is also ok for me. @bzz what do you think?

Reply:
We're changing things to make the app "prod first"... so I'm not sure if it's a good idea to develop new things in the other way...

Reply:
👍  for production-first approach